source/game/field/obj/ObjectShip64.hh

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "game/field/obj/ObjectCollidable.hh"
+#include <egg/math/Vector.hh>
+#include <game/field/ObjectCollisionBase.hh>
+#include <game/field/ObjectCollisionCylinder.hh>
+
+namespace Field {
+
+class ObjectShip64 final : public ObjectCollidable {
+public:
+    ObjectShip64(const System::MapdataGeoObj &params);
+    ~ObjectShip64() = default;
```

Comment:
I don't really know why we fell into this convention, but can you just declare in the header and mark as default in the source? And add a comment above for the address: `/// @addr{0x80765DB0}`